server/controllers/aidControllers.js

```diff
@@ -17,7 +18,12 @@ const Aids = {
                 data: { errors },
             });
         }
+
         const queryText = req.body;
+        const { public_id, url } = await req.file;
+        queryText.imageId = public_id;
+        queryText.imageUrl = url;
```

Comment:
here is a slightly more modern way of righting this
```
const { public_id: imageId, url: imageUrl } = await req.file;
const queryText = { ...req.body, imageId, imageUrl };
```
make the same change in other parts of this code base